### PR TITLE
Broken technical fixes

### PR DESCRIPTION
**Pull request by sdradsb**
In reference to this ticket here: https://github.com/orgs/NASA-IMPACT/projects/29?pane=issue&itemId=30814056

Some technical tutorials do not work and need updates, which is the purpose of this PR.

### CODE REVIEW

File: docs/source/technical_tutorials/visualization/interval_color_mapping.ipynb [hunk 1]
```diff
@@ -2,19 +2,24 @@
  "cells": [
```
Comment (jjfrench) [inline docs/source/technical_tutorials/visualization/interval_color_mapping.ipynb:2]: <p>I don't think this tutorial is making use of pystac-client but it certainly could with a few changes!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (jjfrench) [inline docs/source/technical_tutorials/visualization/interval_color_mapping.ipynb:2]: <pre class="ql-syntax" spellcheck="false">Line #2.    titiler_endpoint = "https://titiler.ops.maap-project.org"
</pre><p>We might want to swap titiler endpoints - titiler-stac.maap-project.org for now?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (jjfrench) [inline docs/source/technical_tutorials/visualization/interval_color_mapping.ipynb:2]: <p>Did we need to change the collection/item on this? I get CRS invalid when trying to pull up the cog/statistics on this item, but the AfriSAR_AGB_Maps_1681___1/Rabi_AGB_50m.tif seems to be working just fine - https://titiler-stac.maap-project.org/cog/statistics?url=s3://nasa-maap-data-store/file-staging/nasa-map/AfriSAR_AGB_Maps_1681___1/Rabi_AGB_50m.tif</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (jjfrench) [inline docs/source/technical_tutorials/visualization/interval_color_mapping.ipynb:2]: <p>Either relates to the titiler endpoint or the item but should probably change both</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (jjfrench) [inline docs/source/technical_tutorials/visualization/interval_color_mapping.ipynb:2]: <p>Run this cell for bin_ranges?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (jjfrench) [inline docs/source/technical_tutorials/visualization/interval_color_mapping.ipynb:2]: <p>Was the title above removed to match guidelines or can we add that back in?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (jjfrench) [inline docs/source/technical_tutorials/visualization/interval_color_mapping.ipynb:2]: <p>Run this cell for color_map? (add back in?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (jjfrench) [inline docs/source/technical_tutorials/visualization/interval_color_mapping.ipynb:2]: <p>Could probably remove tilejson_response and run this cell to output the map</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

File: docs/source/technical_tutorials/access/accessing_data.ipynb [hunk 2]
```diff
@@ -11,7 +11,7 @@
    "cell_type": "markdown",
```
Comment (jjfrench) [inline docs/source/technical_tutorials/access/accessing_data.ipynb:11]: <p>I suggest either merging this cell with the code cell above or moving this code cell before the above markdown cell.</p><p> </p><p>Does this make sense to use the requests library? the getLocalPath() functionality doesn't seem to have gone away. If we're looking to simplify this action, maybe just getData()?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (jjfrench) [inline docs/source/technical_tutorials/access/accessing_data.ipynb:11]: <pre class="ql-syntax" spellcheck="false">Line #9.    maap = MAAP(maap_host='api.ops.maap-project.org')
</pre><p>should remove "ops"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (smk0033) [reply docs/source/technical_tutorials/access/accessing_data.ipynb:11]: <p>Agree that it should be moved up above the markdown or merged with code above</p>

Comment (smk0033) [inline docs/source/technical_tutorials/access/accessing_data.ipynb:11]: <p>My only issue with this cell is I have to remove the [0] - with it, I get an error, but without it I get the OnlineAccessURL.</p><p> </p><p>edit: with the [0] it gives me an error in the old ADE, but not the new one, so this should be fine</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

**Comment by jjfrench** [comment]
@sdradsb can you revert that last commit? I had already committed changes to update that notebook, see https://github.com/MAAP-Project/maap-documentation/pull/253/commits/786260bd942d7ddb32cc78d2d6d2450f8792a735 for the bulk of the changes

**Comment by sdradsb** [comment]
@jjfrench Apologies, didn't notice it. Will revert the commit.

File: docs/source/technical_tutorials/access/accessing_data.ipynb (hunk 2 above)
Comment (jjfrench) [inline docs/source/technical_tutorials/access/accessing_data.ipynb:11]: <p>invalid site link</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

**Comment by jjfrench** [comment]
@smk0033 @sdradsb can I get your reviews on the accessing_cod notebook so we can close this out? 🚀

**Comment by sdradsb** [comment]
Ran all the cells. Looks good to me. Thank you.

**Pull request merged**